src/Tabs/Tabs.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Nav } from '../index'
+
+/**
+ * Tabs Component for Patternfly React
+ */
+const Tabs = props =>
+  <Nav
+    onSelect={props.onSelect}
+    className={props.className}
+    activeKey={props.activeKey}
+    {...props}
+  >
+    {props.children}
+  </Nav>
+
+Tabs.propTypes = {
+  /** tab selected callback */
+  onSelect: PropTypes.func,
+  /** nav tabs class */
+  className: PropTypes.string,
+  /** sets the active tab key */
+  activeKey: PropTypes.number,
+  /** children nodes  */
+  children: PropTypes.node
+}
+Tabs.defaultProps = {
+  className: 'nav nav-tabs'
+}
```

Comment:
For the use case where `<TabRowContents>` would be included to display additional contents (e.g. buttons) on the right side of the tab row, is it possible to conditionally define the default classes to include `pf-tabrow` depending on whether or not `<TabRowContents>` is included? It would be nice to avoid requiring the dev to know that the class `pf-tabrow` needs to be defined if they decide to include.